dependencies/FindGeant4.cmake

```diff
@@ -22,6 +22,13 @@ set_target_properties(geant4
                       PROPERTIES INTERFACE_INCLUDE_DIRECTORIES
                                  "${Geant4_INCLUDE_DIRS}")
 
+# TODO: fix upstream and provide proper variable for library directories
+# for now derive from Geant 4 include dir (first in list)
+list(GET Geant4_INCLUDE_DIRS 0 Geant4_INCLUDE_DIR)
```

Comment:
for my education, is this fixing some problem?

Reply:
@sawenzel Yes, it ensures consistent usage of the settings as detected (and cached) by cmake. In other words, the build is then fully configured by cmake and does not require the Geant4 environment to compile. Together with alisw/alidist#2121 it also fixes recompilation from the BUILD directory without the aliBuild environment.

Do you see an issue with this?

Reply:
no. just wanted to understand the rational.